main.tf

```diff
@@ -1,13 +1,21 @@
-################################################################################
-## defaults
-################################################################################
-terraform {
-  required_version = "~> 1.3, < 2.0.0"
+provider "aws" {
```

Comment:
Provider block should be in root module not in Child.
cc : @tsaucier-sf

Reply:
Done